### ISSUE TITLE
statusbar to support jspreadsheet V9

### ISSUE THREAD

**Issue by dmt0** [GBonnaire/jspreadsheet-plugins-and-editors#10]
### Plugin Name with version

"@jspreadsheet/statusbar": "2.1.0"

### JSpreadsheet version

v9.x

### Describe the bug

As initially raised here: https://github.com/jspreadsheet/pro/issues/225

Can't build app after adding this to plugins:

```
ERROR in ./node_modules/@jspreadsheet/statusbar/index.js 44:18-36
Module not found: Error: Can't resolve 'formula' in '/path/to/my_webapp/node_modules/@jspreadsheet/statusbar'
resolve 'formula' in '/path/to/my_webapp/node_modules/@jspreadsheet/statusbar'
  Parsed request is a module
  using description file: /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/package.json (relative path: .)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      looking for modules in /path/to/my_webapp/src
        single file module
          using description file: /path/to/my_webapp/package.json (relative path: ./src/formula)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/src/formula doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/src/formula.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/src/formula.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/src/formula.wasm doesn't exist
        /path/to/my_webapp/src/formula doesn't exist
      looking for modules in /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/node_modules
        single file module
          using description file: /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/package.json (relative path: ./node_modules/formula)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/node_modules/formula doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/node_modules/formula.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/node_modules/formula.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/node_modules/formula.wasm doesn't exist
        /path/to/my_webapp/node_modules/@jspreadsheet/statusbar/node_modules/formula doesn't exist
      /path/to/my_webapp/node_modules/@jspreadsheet/node_modules doesn't exist or is not a directory
      /path/to/my_webapp/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /path/to/my_webapp/node_modules
        single file module
          using description file: /path/to/my_webapp/package.json (relative path: ./node_modules/formula)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/formula doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/formula.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/formula.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              /path/to/my_webapp/node_modules/formula.wasm doesn't exist
        /path/to/my_webapp/node_modules/formula doesn't exist
      /Users/dima/figlinq/streambed/shelly/webapp/static/node_modules doesn't exist or is not a directory
      /Users/dima/figlinq/streambed/shelly/webapp/node_modules doesn't exist or is not a directory
      /Users/dima/figlinq/streambed/shelly/node_modules doesn't exist or is not a directory
      looking for modules in /Users/dima/figlinq/streambed/node_modules
        single file module
          using description file: /Users/dima/figlinq/streambed/package.json (relative path: ./node_modules/formula)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/dima/figlinq/streambed/node_modules/formula doesn't exist
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/dima/figlinq/streambed/node_modules/formula.js doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/dima/figlinq/streambed/node_modules/formula.json doesn't exist
            .wasm
              Field 'browser' doesn't contain a valid alias configuration
              /Users/dima/figlinq/streambed/node_modules/formula.wasm doesn't exist
        /Users/dima/figlinq/streambed/node_modules/formula doesn't exist
      /Users/dima/figlinq/node_modules doesn't exist or is not a directory
      /Users/dima/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
 @ ./src/workspace/utils/Jspreadsheet.js 8:0-48 40:16-25
 @ ./src/workspace/components/panels/Grid/Grid.js 26:0-55 141:63-92 151:14-42 186:12-40 197:24-55 569:26-38
 @ ./src/workspace/components/controllers/Grid.js 15:0-44 419:29-38
 @ ./src/workspace/reducers/workspace.js 9:0-62 88:26-38
 @ ./src/createStore.js 16:0-62 34:13-29
 @ ./src/app.js 9:0-40 15:14-25

webpack 5.75.0 compiled with 1 error in 18885 ms
```

Line that it complains about:
.../node_modules/@jspreadsheet/statusbar/index.js
![image](https://user-images.githubusercontent.com/2717746/208254771-2584a541-d561-465a-8d81-ef112a0c9eee.png)

Also jspreadsheet V8 got added to my yarn.lock.

### Reproduction steps

```bash
1. yarn add @jspreadsheet/statusbar
2. add it to plugins list
3. try to build
...
broken
```

**Comment by GBonnaire** [comment]
Hi,

Thanks for your ticket. It's fix. You can download last version 2.1.1

**Comment by dmt0** [comment]
Hey @GBonnaire I got 2.1.1, but I see it still lists `jspreadsheet "^8.0.0"` as a dependency. Perhaps it should be a peer dependency?

**Comment by dmt0** [comment]
A couple more observations:
1. It's unstyled:
<img width="1348" alt="image" src="https://user-images.githubusercontent.com/2717746/208544643-c67499af-63a8-485b-b14a-154c65a564af.png">
2. I wonder what I can do to position it below the tabs. Here's the part of my config for tabs:

```
      tabs: {
        allowCreate: true,
        allowChangePosition: false,
        animation: true,
        position: 'bottom',
      },
```

**Comment by GBonnaire** [comment]
Hi, thanks

- I have fixed dependancies, now is `>8.0.0` you can try on 2.1.2
- for style, did you load style.css ? example here : https://github.com/GBonnaire/jspreadsheet-plugins-and-editors/tree/master/plugins/JSSV8/src/statusbar#npm
```javascript
import jss_statusbar from '@jspreadsheet/statusbar';
import '@jspreadsheet/statusbar/style.css';
```

- for change position, after loaded JSS and plugin, you can change position, like this : 

```javascript
function moveUnderJTabsHeader() {
    const jtabsheadersEl = document.querySelector(".jtabs-headers-container");
    const statusbarEl = document.querySelector(".jss_statusbar");
    jtabsheadersEl.after(statusbarEl);
}

jspreadsheet(DOMEl, {
     onload: function() {
           setTimeout(1, moveUnderJTabsHeader);
     }
})
```

**Comment by dmt0** [comment]
Works great, thanks @GBonnaire!

I did this for position change:

```css
.jss-container {
  display: flex !important;
  flex-direction: column !important;
}

.jtabs-header-container {
  order: 3;
}

.jss_statusbar {
  order: 4;
}
```
